src/string_matching.js

```diff
@@ -0,0 +1,34 @@
+import Fuse from "fuse.js";
+
+export function stringMatch(words, searchStr, threshold_score = 0.3, orderedByScope = false) {
```

Comment:
Once you add some tests i think its good to go

Reply:
Added